src/include/ZividPython/Matrix.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include "pybind11/eigen.h"
+#include <Zivid/Matrix.h>
+
+namespace ZividPython::Conversion
+{
+    template<typename T, int rows, int cols>
+    auto toCpp(const Eigen::Matrix<T, rows, cols> &source)
```

Comment:
Change this line to:
`auto toCpp(const Eigen::Matrix<T, rows, cols, Eigen::RowMajor> &source)`